aws/resource_aws_pinpoint_app.go

```diff
@@ -87,20 +88,24 @@ func resourceAwsPinpointApp() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"daily": {
-							Type:     schema.TypeInt,
-							Optional: true,
+							Type:         schema.TypeInt,
```

Comment:
Add validations as described [here](https://docs.aws.amazon.com/pinpoint/latest/apireference/apps-application-id-settings.html#apps-application-id-settings-properties) for `CampaignLimits`.